services/gateway/apis/http-version3/methods/blockchainAppsMetadata.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
 	tags: ['Interoperability'],
 	params: {
 		chainName: { optional: true, type: 'string', min: 1, max: 20, pattern: regex.NAME },
-		chainID: { optional: true, type: 'string', min: 1, max: 21 },
+		chainID: { optional: true, type: 'string', pattern: regex.CHAINID_CSV },
 		network: { optional: true, type: 'string', min: 1, pattern: regex.NETWORK },
```

Comment:
Let us keep the naming patterns consistent:
```suggestion
		chainID: { optional: true, type: 'string', pattern: regex.CHAIN_ID },
		network: { optional: true, type: 'string', min: 1, pattern: regex.NETWORK },
```
or
```suggestion
		chainID: { optional: true, type: 'string', pattern: regex.CHAIN_ID_CSV },
		network: { optional: true, type: 'string', min: 1, pattern: regex.NETWORK_CSV },
```

Reply:
Applied.